src/groundlight/client.py

```diff
@@ -249,3 +265,22 @@ def add_label(self, image_query: Union[ImageQuery, str], label: Union[Label, str
         api_label = convert_display_label_to_internal(image_query_id, label)
 
         return self.api_client._add_label(image_query_id, api_label)  # pylint: disable=protected-access
+
+    def start_inspection(self) -> str:
+        """Starts an inspection report and returns the id of the inspection."""
+        return self.api_client.start_inspection()
+
+    def update_inspection_metadata(self, inspection_id: str, user_provided_key, user_provided_value) -> None:
```

Comment:
Let's add type-hints on `user_provided_*`.  I think they're both str, right?

Reply:
Yes, they are. I hesitated about this initially, because the code I was copying from the RPC server did not have type hints, but they will be treated as strings in the end, so I will add the type hints.